infoschema/metric_schema.go

```diff
@@ -925,4 +925,1055 @@ var MetricTableMap = map[string]MetricTableDef{
+		Labels:  []string{"instance", "type"},
+		Comment: "The total number of the gRPC message failures",
+	},
+	"tikv_region_count": {
+		PromQL:  `sum(tikv_raftstore_region_count{$LABEL_CONDITIONS}) by (instance,type)`,
+		Labels:  []string{"instance", "type"},
+		Comment: "The number of regions on each TiKV instance",
+	},
+	"tikv_scheduler_is_busy": {
+		PromQL:  `sum(rate(tikv_scheduler_too_busy_total{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance,db,type,stage)`,
+		Labels:  []string{"db", "instance", "type", "stage"},
+		Comment: "Indicates occurrences of events that make the TiKV instance unavailable temporarily, such as Scheduler Busy",
+	},
+	"tikv_channel_full_total": {
+		PromQL:  `sum(rate(tikv_channel_full_total{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance,type,db)`,
+		Labels:  []string{"db", "instance", "type"},
+		Comment: "The total number of channel full errors on each TiKV instance, it will make the TiKV instance unavailable temporarily",
+	},
+	"tikv_coprocessor_is_busy": {
+		PromQL:  `sum(rate(tikv_coprocessor_request_error{type='full'}[$RANGE_DURATION])) by (instance,db,type)`,
+		Labels:  []string{"db", "instance"},
+		Comment: "Indicates occurrences of events that make the TiKV instance unavailable temporarily, such as Coprocessor Full",
+	},
+	"tikv_engine_write_stall": {
+		PromQL:  `avg(tikv_engine_write_stall{type="write_stall_percentile99"}) by (instance, db)`,
+		Labels:  []string{"db", "instance"},
+		Comment: "Indicates occurrences of events that make the TiKV instance unavailable temporarily, such as Write Stall",
+	},
+	"tikv_server_report_failures": {
+		PromQL:  `sum(rate(tikv_server_report_failure_msg_total{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance,store_id)`,
+		Labels:  []string{"instance", "store_id", "type"},
+		Comment: "The total number of reporting failure messages",
+	},
+	"tikv_storage_async_requests": {
+		PromQL:  `sum(rate(tikv_storage_engine_async_request_total{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance, status, type)`,
+		Labels:  []string{"instance", "status", "type"},
+		Comment: "The number of different raftstore errors on each TiKV instance",
+	},
+	"tikv_scheduler_stage_total": {
+		PromQL:  `sum(rate(tikv_scheduler_stage_total{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance, stage,type)`,
+		Labels:  []string{"instance", "stage", "type"},
+		Comment: "The number of scheduler state on each TiKV instance",
+	},
+	"tikv_coprocessor_request_error": {
+		PromQL:  `sum(rate(tikv_coprocessor_request_error{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance, reason)`,
+		Labels:  []string{"instance", "reason"},
+		Comment: "The number of different coprocessor errors on each TiKV instance",
+	},
+	"tikv_region_change": {
+		PromQL:  `sum(delta(tikv_raftstore_region_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance,type)`,
+		Labels:  []string{"instance", "type"},
+		Comment: "The count of region change per TiKV instance",
```

Comment:
Is it number of regions or occurrences of region splits/merges?

Reply:
No, `tikv_raftstore_region_count` just indicates the region count.
The promQL `sum(delta...` indicates the region count change per TiKV instances.